### ISSUE TITLE
Unable to download the SimpleCopy V2025.3 from Add-on Manager

### ISSUE THREAD

**Issue by chaichaimee** [nvaccess/addon-datastore#6051]
Hello,
I would like to report that the SimpleCopy add-on (version 2025.3) is visible in the NVDA Add-on Manager (Add-on Store), but when I try to download and install it, an error occurs stating that the add-on cannot be downloaded.
Here is the direct download link specified for this version:
🔗 https://github.com/chaichaimee/simpleCopy/releases/download/v2025.3/simpleCopy-v2025.3.nvda-addon
The full project repository is here:
🔗 https://github.com/chaichaimee/simpleCopy
I have verified that:
✅ The .nvda-addon file exists at the release link.
✅ The release is public.
✅ The link works when accessed manually via a web browser.
Despite this, the Add-on Manager in NVDA is unable to download the add-on automatically.
Could you please check if there is a problem with how the download link is configured in the addon-datastore repository, or if there is a caching issue within the Add-on Store system?
Thank you very much for your assistance.

**Comment by seanbudd** [comment]
Please don't submit the same add-on 100 times.
You need to create a new version to update the addon (i.e. increase the version number). I
Each submitted addon link must be static, i.e. it must always download the exact same file with the exact same addon hash.
f you change the file which is downloaded from previous addon submissions it will not be able to be downloaded and will eventually be removed from the store.

As you changed the file in the release, it will no longer be downloaded in the add-on store, as the file hash is changed.

In future, stop resubmitting an add-on over and over if the first submission doesn't work.
